apps/dashboard/app/views/projects/import.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='page-header text-center pb-3'>
+  <h1>
+    Import a Shared Project
+  </h1>
+</div>
+
+<%= bootstrap_form_for(@project, url: project_import_save_path, method: :post) do |form| %>
+  <%= render partial: "form", locals: { form: form }%>
+<% end %>
```

Comment:
I think we need a new partial for this. I'm thinking it'll look a lot like the index with a list of cards. Maybe there's a simple search? Maybe you can expand the cards to see more details? 

In any case, I'd say let's just put a TODO here and reset the `_form.html.erb`.